src/main/java/com/uid2/admin/auth/OktaAuthProvider.java

```diff
@@ -11,16 +12,34 @@
 import com.okta.jwt.JwtVerifiers;
 import static com.uid2.admin.auth.AuthUtil.isAuthDisabled;
 import java.time.Duration;
-import java.util.Arrays;
 import java.util.List;
-import java.util.Collections;
 
 public class OktaAuthProvider implements AuthProvider {
     private final JsonObject config;
-    private List<String> scopes = Collections.unmodifiableList(Arrays.asList("openid",  "profile" ,"email" ,"groups"));
-
+    private final List<String> scopes = List.of("openid", "profile", "email", "uid2.admin.human", "offline_access");
+    private final AccessTokenVerifier accessTokenVerifier;
+    private final IdTokenVerifier idTokenVerifier;
     public OktaAuthProvider(JsonObject config) {
         this.config = config;
+        if(isAuthDisabled(config)) {
+            this.accessTokenVerifier = null;
+            this.idTokenVerifier = null;
+            return;
+        }
+        this.accessTokenVerifier = JwtVerifiers.accessTokenVerifierBuilder()
+                .setIssuer(config.getString("okta_auth_server"))
```

Comment:
might be better to declare string const for all okta config as they are being used in multiple places